src/libsyntax/feature_gate.rs

```diff
@@ -1625,7 +1625,7 @@ impl<'a> Visitor<'a> for PostExpansionVisitor<'a> {
             }
 
             ast::ItemKind::Struct(..) => {
-                if let Some(attr) = attr::find_by_name(&i.attrs[..], "repr") {
+                for attr in attr::filter_by_name(&i.attrs[..], "repr") {
```

Comment:
This makes me wonder in how many other cases we have the same problem...

Reply:
This was the only one that looked like a problem for `repr`. I don't know about other attributes though.